### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by patricktcoakley**
#42 

Only an owner can change the GitHub repo description, which still says: 

> Embrace the capabilities of Blazor on .NET 8 to craft static websites.

I did a few minor edits to the README, let me know if you want to change anything but it was mostly just small things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,26 +9,25 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/4fa2c17a-6385-4cc6-9919-e32c134175d9/deploy-status)](https://app.netlify.com/sites/blazorstatic/deploys)
 
 
-Embrace the capabilities of Blazor on .NET 8 to craft static websites.
```
Comment (tesar-tech) [inline README.md:12]: I would remove the "on .NET" part. There is no Blazor without dotnet.  Or maybe leave out Blazor and keep the .NET. Blazor is in the name and .NET is known better (?). Also it makes it shorter. 

Base on your decision, I will change the description.

File: README.md [hunk 2]
```diff
@@ -9,26 +9,25 @@
...
 
-Like any .NET Blazor app! That's the main point. If you know how to build a Blazor app, you can transfer it to HTML files easily.
-**BlazorStatic** will help you with that. It can also help with processing markdown files.
+**BlazorStatic** aims to be intuitive and easy to use for most developers who have worked with any kind of front-end framework, especially if you already work with Blazor.
 
-> Website for BlazorStatic is a feature-rich example of the library. Check the [source code](https://github.com/tesar-tech/BlazorStatic/tree/master/BlazorStaticWebsite). 
+> The site for BlazorStatic is a feature-rich example of the library. Check the [source code](https://github.com/tesar-tech/BlazorStatic/tree/master/BlazorStaticWebsite). 
 
-Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, which you can deploy to any static hosting service.
+Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, and from there you can deploy to any standard HTTP server or static site hosting service.
```
Comment (tesar-tech) [inline README.md]: what about: "to any server or hosting"?

File: README.md [hunk 3]
```diff
@@ -9,26 +9,25 @@
...
 
 There are many defaults to keep usage simple, but you can configure it extensively. For example, you can match your YAML front matter in markdown files (which can be reused from other static site generators).
 
 
 
 # 🚀 Getting Started
 
-You can start using BlazorStatic in three ways, depending on your setup and intended use. 
+You can start using BlazorStatic in three ways, depending on your setup and intended use. Regardless of which way you choose, you'll need at least [.NET SDK 8](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) installed.
```
Comment (tesar-tech) [inline README.md]: Please make it shorter. "At least .NET 8 is required." (??)

**Review by tesar-tech — CHANGES_REQUESTED** [review]
Few issue. I am just trying to make it shorter and more condense. But I admit it's rather my personal preference and I am not native English speaker, so I might be wrong on some places.

File: README.md (hunk 2 above)
Comment (patricktcoakley) [reply README.md]: I think it's nice to make it clear that it's just a bundle of files that an HTTP server could host, but I don't really have a strong opinion either way.

File: README.md (hunk 1 above)
Comment (patricktcoakley) [reply README.md:12]: Yeah I can remove the .NET. What about "Embrace the power" instead of "capabilities"?

Comment (tesar-tech) [reply README.md:12]: yes, that is also something that was meant for .net 8 as the capabilities were not present prior..

File: .DS_Store
Comment (tesar-tech) [inline .DS_Store:1]: Please remove the .ds_store (and add it to .gitignore for the future)

File: README.md (hunk 2 above)
Comment (tesar-tech) [reply README.md]: for me it's hard to figure out what "standart HTTP server" means. It is probably technically correct, but doesn't imply the solution is easy..

```md
Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, ready to be [deployed almost anywhere](https://tesar-tech.github.io/BlazorStatic/docs/deployment), often for free.
```

Comment (patricktcoakley) [reply README.md]: Any HTTP server (nginx, caddy, Apache) can host your files. You are not limited to a hosting service, which is part of why SSG is so popular: it's simple. The current documentation implies that you need to "deploy" it somewhere particular, like the cloud, which is not the case - I could host my files locally at home, on a VPS, on a VM, or anywhere.

**Pull request merged**